.github/workflows/release.yml

```diff
@@ -11,6 +11,43 @@ jobs:
   release:
     name: release ${{ github.event.ref }}
     needs: before-release
-    uses: ansible-community/devtools/.github/workflows/release.yml@main
-    with:
-      pypi: true
+    # unable to use environment with uses/with, basically cannot reuse release pipelines
```

Comment:
do you have a demo of this?

Reply:
Try to add using the visual editor and you will see a nice error in the web editor. No need to run it, they added it to the live editor.

I am not sure if the called workflow can make use of its own environment secrets but if it does it would be big security issue as it would mean that callers would be able to use its secrets. Anyway we would not be able to re-use it as we have several accounts, one for each organization. Shortly, that seems to be a dead end.

Feel free to experiment with others ways of reusing the release pipelines, maybe we will find a solution. still, for the time being I am reverting to the old / known-to-work way. Lucky, I think that is the only repo where I attempted to reuse release pipeline.